EventFilter/CSCRawToDigi/plugins/CSCViewDigi.cc

```diff
@@ -22,8 +22,10 @@
 #include "DataFormats/CSCDigi/interface/CSCDCCFormatStatusDigiCollection.h"
 #include "DataFormats/CSCDigi/interface/CSCDDUStatusDigiCollection.h"
 #include "DataFormats/CSCDigi/interface/CSCDCCStatusDigiCollection.h"
+#include "DataFormats/CSCDigi/interface/CSCShowerDigiCollection.h"
+#include "DataFormats/GEMDigi/interface/GEMPadDigiClusterCollection.h"
 
-class CSCViewDigi : public edm::EDAnalyzer {
+class CSCViewDigi : public edm::one::EDAnalyzer<> {
```

Comment:
Same here, it could become `global`:  again. it does not seem to be used in production, therefore there is no need to update it.